### ISSUE TITLE
HMTL snippets in historical call

### ISSUE THREAD

**Issue by bushrang3r** [lukaszbanasiak/yahoo-finance#130]
Trying yahoo finance for the first time today and getting a bunch of HTML information when calling get_historical.

`>>> pprint(ann.get_historical('2017-05-02','2017-05-05'))
[{u'Symbol': u'ANN.AX', u'_HTML_': u'<HEAD>'},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'<TITLE>Error</TITLE>'},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'</HEAD>'},
 {u'Symbol': u'ANN.AX', u'_HTML_': None},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'<BODY BGCOLOR="white" FGCOLOR="black">'},
 {u'Symbol': u'ANN.AX', u'_HTML_': u''},
 {u'Symbol': u'ANN.AX', u'_HTML_': u''},
 {u'Symbol': u'ANN.AX',
  u'_HTML_': u''},
 {u'Symbol': u'ANN.AX', u'_HTML_': u''},
 {u'Symbol': u'ANN.AX',
  u'_HTML_': u''},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'<H1>Error</H1>'},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'<HR>'},
 {u'Symbol': u'ANN.AX', u'_HTML_': None},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'<FONT FACE="Helvetica,Arial"><B>'},
 {u'Symbol': u'ANN.AX',
  u'_HTML_': u'Description: Could not process this "GET" request.'},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'</B></FONT>'},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'<HR>'},
 {u'Symbol': u'ANN.AX', u'_HTML_': u'</BODY>'}]`

**Comment by DuckBoss** [comment]
^ I am getting this issue recently too.

**Comment by torshak-mozyora** [comment]
please fix this issue as yahoo has stopped its finance api(or changed something don't know,,, but another GitHub user ranaroussi seems to have found a way --- yahoo-finale-fix.

but this only works for us stocks please make some amendment so that indian stocks markets are also compatible NSE and BOM

**Comment by taymourheikal** [comment]
Bump

**Comment by geoeye123** [comment]
Yes, ranaroussi fixed this issue with the yahoo-finance-fix.  Can be installed via pip or similar, and download() method replaces the get_historical() from yahoo_finance.  Can be found here:
https://github.com/ranaroussi/fix-yahoo-finance
